frontend/README.md

```diff
@@ -140,3 +138,50 @@ swagger-codegen-cli@2.4.7, which you can get
 [here](https://repo1.maven.org/maven2/io/swagger/swagger-codegen-cli/2.4.7/).
 Make sure the jar file is somewhere on your path with the name
 swagger-codegen-cli.jar, then run `npm run apis`.
+
+## MLMD components
+
+* `src/mlmd` - components for visualizing data from an `ml-metadata` store. For more information see the
+ [kubeflow/metadata](https://github.com/kubeflow/metadata) repository.
+
+This module previously lived in [kubeflow/frontend](https://github.com/kubeflow/frontend) repository. It contains tsx files and `/generated/ml_metadata` for MLMD protos.
+
+### Building generated metadata Protocol Buffers
+
+* `build:protos` - for compiling Protocol Buffer definitions
+
+
+This project contains a mix of natively defined classes and classes generated by the Protocol
+Buffer Compiler from definitions in the `proto/` directory. Copies of the generated classes are
+included in the `src/mlmd` directory to allow the build process to succeed without a dependency on
+the Protocol Buffer compiler, `protoc`, being in the system PATH.
```

Comment:
TODO: after updating paths, refresh paths in the README

Reply:
Sounds good, updated.